package.json

```diff
@@ -52,6 +52,38 @@
       }
     }
   },
+  "consumedServices": {
+    "runInTerminal": {
+      "description": "Allow to run commands in platformio-ide-terminal.",
+      "versions": {
+        "0.14.5": "consumeTerminal"
+      }
+    },
+    "platformioIDETerminal": {
+      "description": "Allow to run commands in platformio-ide-terminal.",
+      "versions": {
+        "^1.1.0": "consumeTerminal"
+      }
+    },
+    "terminusTerminal": {
+      "description": "Allow to run commands in terminus.",
+      "versions": {
+        "^1.1.1": "consumeTerminal"
+      }
+    },
+    "terminationTerminal": {
+      "description": "Allow to run commands in termination.",
+      "versions": {
+        "^1.1.0": "consumeTerminal"
+      }
+    },
+    "terminal": {
```

Comment:
This allow almost any terminal including old ones that haven't been updated in a while. If we only want currently maintained terminals we can just consume `terminal`.

Reply:
It doesn't hurt to support them if this doesn't break other things. Regarding package curation, I have some good plans which will solve many issues.